website/blog/2023-01-11-databend-weekly-76.md

```diff
@@ -0,0 +1,155 @@
+---
+title: 'This Week in Databend #76'
+date: 2023-01-11
+slug: 2023-01-11-databend-weekly
+tags: [databend, weekly]
+authors:
+- name: PsiACE
+  url: https://github.com/psiace
+  image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a powerful cloud data warehouse. Built for elasticity and efficiency. Free and open. Also available in the cloud: <https://app.databend.com> .
+
+## What's New
+
+Check out what we've done this week to make Databend even better for you.
+
+### Features & Improvements :sparkles:
+
+**Meta**
+
+- add reader-min-msg-ver and msg-min-reader-ver in proto-conv ([#9535](https://github.com/datafuselabs/databend/pull/9535))
+
+**Planner**
+
+- support `tuple.1` and `get(1)(tuple)` ([#9493](https://github.com/datafuselabs/databend/pull/9493))
+- support display estimated rows in `EXPLAIN` ([#9528](https://github.com/datafuselabs/databend/pull/9528))
+
+**Query**
+
+- efficiently memory two level group by in standalone mode ([#9504](https://github.com/datafuselabs/databend/pull/9504))
+
+**Storage**
+
+- support nested type in `read_parquet` ([#9486](https://github.com/datafuselabs/databend/pull/9486))
+- add build options table ([#9502](https://github.com/datafuselabs/databend/pull/9502))
+
+### Code Refactoring :tada:
+
+**\***
+
+- merge new expression ([#9411](https://github.com/datafuselabs/databend/pull/9411))
+- remove and rename crates ([#9481](https://github.com/datafuselabs/databend/pull/9481))
+- bump rust version ([#9540](https://github.com/datafuselabs/databend/pull/9540))
+
+**Expression**
+
+- move negative funtions to binder ([#9484](https://github.com/datafuselabs/databend/pull/9484))
+- use error_to_null() to eval try_cast ([#9545](https://github.com/datafuselabs/databend/pull/9545))
+  
+**Functions**
+
+- replace h3ron to h3o ([#9553](https://github.com/datafuselabs/databend/pull/9553))
+
+**Format**
+
+- extract `AligningStateTextBased` ([#9472](https://github.com/datafuselabs/databend/pull/9472))
+- richer error context ([#9534](https://github.com/datafuselabs/databend/pull/9534))
+
+**Query**
+
+- use ctx to store the function evaluation error ([#9501](https://github.com/datafuselabs/databend/pull/9501))
+- refactor map access to support view read tuple inner ([#9516](https://github.com/datafuselabs/databend/pull/9516))
+
+**Storage**
+
+- bump opendal for streaming read support ([#9503](https://github.com/datafuselabs/databend/pull/9503))
+- refactor bloom index to use vectorized siphash function ([#9542](https://github.com/datafuselabs/databend/pull/9542))
+
+### Bug Fixes :wrench:
+
+**HashTable**
+
+- fix memory leak for unsized hash table ([#9551](https://github.com/datafuselabs/databend/pull/9551))
+
+**Storage**
+
+- fix row group stats collection ([#9537](https://github.com/datafuselabs/databend/pull/9537))
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Databend Now Works with New Expression
```

Comment:
### New Year, New Expression!

We're so thrilled to tell you that Databend now fully works with New Expression after more than a half year of dedicated work. New Expression introduces a formal type system to Databend and supports type-safe downward casting, making the definition of functions easier.

New Expression is still being tuned, and a new version (v0.9) of Databend will be released once the tuning work is complete.